### ISSUE TITLE
TAGS section set\get functions

### ISSUE THREAD

**Issue by eladsal** [OpenWaterAnalytics/EPANET#631]
I thought we already covered this section but I don't see these functions. I think this section is not parsed at all.

**Comment by LRossman** [comment]
That is correct. The only processing the `[TAGS]` section of an input file receives is when `ENsaveinpfile` is called to save a network's data. In that case it merely copies that section from the existing `INP `file to the new one.

**Comment by eladsal** [comment]
Should we keep this issue open for future enhancement?

**Comment by Mariosmsk** [comment]
I think yes, I guess you mean the section [LABELS]. I think it is also good to include the section [BACKDROP]

**Comment by eladsal** [comment]
@Mariosmsk I was actually referring to the `[TAGS]` section, see here:
https://epanet22.readthedocs.io/en/latest/input_keywords.html

**Comment by LRossman** [comment]
I've implemented the set/get TAGS feature in branch `dev-tags`. It includes a unit test added into _test_node.cpp_. If interested, please have a look and provide any feedback before I submit a PR (quite a few files had to be updated).

I also took the opportunity to re-format the Release Notes file to make it easier to read.
